tests/test-cluster.py

```diff
@@ -53,16 +56,16 @@ def setup(self, channel_or_snap):
         if (path.exists("/snap/bin/multipass") and not self.backend) or self.backend == "multipass":
             print("Creating mulitpass VM")
             self.backend = "multipass"
-            self._setup_multipass(channel_or_snap)
+            self._setup_multipass(channel_or_snap, launch_configs)
```

Comment:
Where is the multipass setup update that takes into account this variable? I only see the lxc one.